torf/_torrent.pyi

```diff
@@ -14,10 +13,6 @@ from ._errors import TorfError
 from ._magnet import Magnet
 from ._utils import File, Filepath, Filepaths, Files, MonitoredList, Trackers, URLs
 
-if sys.version_info < (3, 11):
-    from typing_extensions import NotRequired, Required, TypedDict
```

Comment:
I'm sorry if I wasn't clear enough, but you still need to import from it because it's not present in python < 3.11. You just don't have to *depend* on typing_extensions because type checkers will assume it's installed.

Reply:
oop. ok reverted